### ISSUE TITLE
dash workaround

### ISSUE THREAD

**Issue by heiskane** [calebstewart/pwncat#28]
I was connected to a debian target machine and got the issues mentioned in the readme file but another workaroud i found is to spawn a /bin/bash with the usual python trick on the target and then setting the prompt to fancy. This workaround makes all the nice and fancy colors work. ps the 'python trick' im referring to is ```python3 -c 'import pty;pty.spawn("/bin/bash")'```

**Comment by calebstewart** [comment]
Yes, spawning a different shell would be a work-around. The intent was to not depend directly on bash being installed. Also, this will technically work with any other "user-focused" shell. I.e. you could also solve the problem by running `zsh`. There's a long list of shells that handle the "fancy" output properly so it would end up being a long list of options to check in order to be "comprehensive", which could slow things down depending on the machine you connect to. Because of this, I opted to not implement these checks. Currently, `pwncat` checks what shell is currently being used, and re-uses that shell path for any new shells that it needs, so it doesn't need to figure out every shell that is available.

Bottom line: I'm open to other solutions beyond what is done currently, but in my mind "just running another shell" breaks the "live of the land" methodology that I was going for, and checking the availability of a potentially long list of other shells could cause performance issues.
